locations/spiders/dunkin.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import re
+from locations.items import GeojsonPointItem
+import json
+
+class DunkinSpider(scrapy.Spider):
+    name = "dunkindonuts"
+    allowed_domains = ["dunkindonuts.com","mapquestapi.com"]
+
+    # Choosing key hub at geographic extremes of the US in order to get full coverage in the location search
+    search_hubs = ['washington+dc','los+angeles','miami','boston','chicago']
+
+    start_urls = ['https://www.mapquestapi.com/search/v2/radius?callback=' \
+                  'jQuery111205309142260006529_1512766475273&key=Gmjtd|lu6t2l' \
```

Comment:
Remove the `callback=...` parameter here and you won't have to do the `pure_query` regex search down below.